Education/Blogs.md

```diff
@@ -1,2 +1,27 @@
-## Blogs
+To ensure we remain up to date with our technologies and stay on the edge of them we often come up to interesting topics, articles and discussions. If you bump into something interesting or unusual on the project you are working on or have something else you would like to discuss feel free to mention it on our weekly meeting. If the weekly meeting cannot wait, you can always share it and comment it in our main Slack channel `#team-dotnet`.
+Additionally every now and then you will find some article or a blogpost that will grab your attention. If you think it would benefit our team to give it a read, feel free to share it in our Slack newsletter channel: `#newsletter-dotnet`. 
+### Blogs
```

Comment:
I'm not sure about the titles, but when you don't put an empty line between the bullet points and other text, the formatting tends to break